internal/cli/watch_opts.go

```diff
@@ -29,7 +29,7 @@ type WatchOpts struct {
 	s              *spinner.Spinner
 	EnableWatch    bool
 	DefaultWait    time.Duration
-	Timeout        uint
```

Comment:
[q] is `uint` discouraged now?

Reply:
The issue is that the places where it's used (like [here](https://github.com/mongodb/mongodb-atlas-cli/blob/46ea2ba44076b660fd3c8eb2e2568a75f10f0958/internal/cli/clusters/create.go#L125)), we always convert from `uint` to `time.Duration`, which is `int64`. Linting complains about possible overflows.

Example usage
```go
watcher.Timeout = time.Duration(opts.Timeout)
```